Flaky TestWasbBlobSensorTrigger test

````
FAILED providers/microsoft/azure/tests/unit/microsoft/azure/triggers/test_wasb.py::TestWasbBlobSensorTrigger::test_success - AssertionError: assert False is True
 +  where False = <built-in method done of _asyncio.Task object at 0xffe8138b6b00>()
 +    where <built-in method done of _asyncio.Task object at 0xffe8138b6b00> = <Task pending name='Task-2479' coro=<<async_generator_asend without __name__>()> wait_for=<Future pending cb=[shield.<locals>._outer_done_callback() at /usr/python/lib/python3.12/asyncio/tasks.py:922, Task.task_wakeup()]>>.done
```

For example https://github.com/apache/airflow/actions/runs/17534123692/job/49795194833#step:8:2322

--- a/providers/microsoft/azure/tests/unit/microsoft/azure/triggers/test_wasb.py
+++ b/providers/microsoft/azure/tests/unit/microsoft/azure/triggers/test_wasb.py
@@ -23,6 +23,7 @@ from unittest import mock
 
 import pytest
 
+from airflow.models import Connection
 from airflow.providers.microsoft.azure.triggers.wasb import (
     WasbBlobSensorTrigger,
     WasbPrefixSensorTrigger,
@@ -35,7 +36,17 @@ TEST_DATA_STORAGE_BLOB_PREFIX = TEST_DATA_STORAGE_BLOB_NAME[:10]
 TEST_WASB_CONN_ID = "wasb_default"
 POKE_INTERVAL = 5.0
 
-pytestmark = pytest.mark.db_test
+
+@pytest.fixture(autouse=True)
+def setup_connections(create_connection_without_db):
+    create_connection_without_db(
+        Connection(
+            conn_id=TEST_WASB_CONN_ID,
+            conn_type="wasb",
+            login="test_account_name",
+            password="test_account_key",
+        )
+    )
 
 
 class TestWasbBlobSensorTrigger:

